superdesk/errors.py

```diff
@@ -733,20 +733,26 @@ def __init__(self, errors, fields, message=None):
         Exception.__init__(self)
         self.errors = errors
         self.fields = fields
-        self.response = send_response(
-            None, (
-                {
-                    app.config['STATUS']: app.config['STATUS_ERR'],
-                    app.config['ISSUES']: {
-                        'validator exception': str([self.errors]),  # BC
-                        'fields': self.fields,
+        try:
+            self.response = send_response(
+                None, (
+                    {
+                        app.config['STATUS']: app.config['STATUS_ERR'],
+                        app.config['ISSUES']: {
+                            'validator exception': str([self.errors]),  # BC
+                            'fields': self.fields,
+                        },
                     },
-                },
-                None,
-                None,
-                400,
+                    None,
+                    None,
+                    400,
+                )
             )
-        )
+        except RuntimeError as e:
+            # the exception is run outside of request context
+            # it may be the case with CLI commands
+            # we log the error to not loose the initial error
+            logger.warning(e)
```

Comment:
given that it's an error - why not use `logger.error()` ?

Reply:
because in the case of CLI command, it's not an error. There error is actually in `SuperdeskValidationError` assuming that we are in an HTTP Request. I was initially wanting to pass it and not log anything, but a warning is still useful, as something may be wrong. So IMHO a warning is adapted here, not an error.

Reply:
ok makes sense